src/tic_tac_toe/player.clj

```diff
@@ -0,0 +1,6 @@
+(ns tic-tac-toe.player
+  (:use [tic-tac-toe.user-interface :as ui]))
+
+(defn pick-move
+  []
+  (- (ui/parse-int (ui/get-input)) 1))
```

Comment:
Exposing `parse-int` from the UI module feels like a violation of SRP to me.

Reply:
Removed this from the player.  Eventually wanted this to go to some kind of validator in a future story.  For now, I have made a very simple [referee](https://github.com/saralein/tic-tac-toe_clojure/pull/1/files#diff-f014d0e7db79dee6fc42af72f74d5325R1) to handle converting to integer and converting to a spot (which is a board index).  I will incorporate validation into this referee in the validation story in the future.

Reply:
Using vernacular that describes what a things does, rather than the domain its related to makes me very happy. I really like "referee" over "rules"